pkg/promlib/healthcheck.go

```diff
@@ -16,11 +16,8 @@ const (
 	refID = "__healthcheck__"
 )
 
-var logger = backend.NewLoggerWith("logger", "tsdb.prometheus")
-
 func (s *Service) CheckHealth(ctx context.Context, req *backend.CheckHealthRequest) (*backend.CheckHealthResult,
 	error) {
-	logger := logger.FromContext(ctx)
```

Comment:
you are losing the contextual information here. You probably want to do:

```
logger := s.logger.FromContext(ctx)
```

Reply:
https://github.com/grafana/grafana/pull/84463/commits/8b2e8e8dce2799b140f91f1f36741374a9ca2b98